records_versioning/services.py

```
@@ -23,23 +23,21 @@ def _compute_snapshot(instance: object) -> dict[str, object]:
     
     Returns:
-        dict: دیکشنریِ فیلدها و مقادیرِ نمونه که UUIDها به رشته تبدیل شده‌اند.
+        dict: دیکشنریِ فیلدها و مقادیرِ نمونه که datetime ها به رشته تبدیل شده‌اند.
     """
     d = model_to_dict(instance)
-    # Convert UUIDs to strings for JSON serialization
+    # Convert datetime and date objects to strings for JSON serialization
     for key, value in d.items():
-        if isinstance(value, uuid.UUID):
-            d[key] = str(value)
+        if isinstance(value, (datetime, date)):
```

Ruff (0.12.2)

Use `X | Y` in `isinstance` call instead of `(X, Y)`

Convert to `X | Y`

(UP038)